.github/workflows/ci.yml

```diff
@@ -35,6 +35,13 @@ jobs:
       - uses: asdf-vm/actions/install@v1
         if: steps.asdf-cache.outputs.cache-hit != 'true'
       - uses: mbta/actions/reshim-asdf@v1
+        if: steps.asdf-cache.outputs.cache-hit == 'true'
```

Comment:
You always need to do this, as it also sets up the `PATH` environment variable to point at the ASDF versions of Elixir/Erlang.

Reply:
Oh, I didn't expect we'd have to, since I assumed `asdf-vm/actions/install` would fully set things up. Maybe we should name our action something other than `reshim-asdf` since it does more than just reshim?